src/openrct2/object/RideObject.cpp

```diff
@@ -805,39 +808,72 @@ rct_ride_entry_vehicle RideObject::ReadJsonCar(json_t& jCar)
                 { "corkscrews", VEHICLE_SPRITE_FLAG_CORKSCREWS },
                 { "restraintAnimation", VEHICLE_SPRITE_FLAG_RESTRAINT_ANIMATION },
                 { "curvedLiftHill", VEHICLE_SPRITE_FLAG_CURVED_LIFT_HILL },
-                { "VEHICLE_SPRITE_FLAG_15", VEHICLE_SPRITE_FLAG_15 },
+                { "VEHICLE_SPRITE_FLAG_15", VEHICLE_SPRITE_FLAG_USE_4_ROTATION_FRAMES },
             });
     }
 
     car.flags |= Json::GetFlags<uint32_t>(
         jCar,
         {
+            { "isPoweredRideWithUnrestrictedGravity", VEHICLE_ENTRY_FLAG_POWERED_RIDE_UNRESTRICTED_GRAVITY },
+            { "hasNoUpstopWheels", VEHICLE_ENTRY_FLAG_NO_UPSTOP_WHEELS },
+            { "hasNoUpstopBobsleigh", VEHICLE_ENTRY_FLAG_NO_UPSTOP_BOBSLEIGH },
+            { "isMiniGolf", VEHICLE_ENTRY_FLAG_MINI_GOLF },
+            { "isReverserBogie", VEHICLE_ENTRY_FLAG_REVERSER_BOGIE },
+            { "isReverserPassengerCar", VEHICLE_ENTRY_FLAG_REVERSER_PASSENGER_CAR },
+            { "hasInvertedSpriteSet", VEHICLE_ENTRY_FLAG_HAS_INVERTED_SPRITE_SET },
+            { "hasDodgemInUseLights", VEHICLE_ENTRY_FLAG_DODGEM_INUSE_LIGHTS },
+            { "allowDoorsDeprecated", VEHICLE_ENTRY_FLAG_ALLOW_DOORS_DEPRECATED },
```

Comment:
Get rid of this line - we don't use this flag any more. Also get rid of the "old line".

Reply:
The old one is still in [OpenRCT2/objects](https://github.com/OpenRCT2/objects/search?q=VEHICLE_ENTRY_FLAG_ALLOW_DOORS_DEPRECATED) though. Will the game be find if the flag cannot be found?

Reply:
Update: the game doesn't show any errors and Haunted Harbor still looks good, so it seems fine to remove the old one too. :)